src/Sdk/DTGenerated/Generated/TaskAgentHttpClientBase.cs

```diff
@@ -703,6 +704,33 @@ public TaskAgentHttpClientBase(Uri baseUrl, HttpMessageHandler pipeline, bool di
                 cancellationToken: cancellationToken);
         }
 
+        /// <summary>
+        /// [Preview API]
+        /// </summary>
+        /// <param name="id">Message ID</param>
+        /// <param name="userState"></param>
+        /// <param name="cancellationToken">The cancellation token to cancel operation.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public virtual Task<Pipelines.AgentJobRequestMessage> GetJobMessageAsync(
+            string id,
+            object userState = null,
+            CancellationToken cancellationToken = default)
+        {
+            HttpMethod httpMethod = new HttpMethod("GET");
+            Guid locationId = new Guid("25adab70-1379-4186-be8e-b643061ebe3a");
+
+            List<KeyValuePair<string, string>> queryParams = new List<KeyValuePair<string, string>>();
+            queryParams.Add("id", id);
```

Comment:
it's a little bit weird that we are using query parameters instead of PATH. 😄

Reply:
good catch.... had a bunch of query params early when testing. In the process of switching to message ID as the only parameter now. Will switch to a value in the path. I like that.